solutions/landing-zone-v2/namespaces/hierarchy.yaml

```diff
@@ -112,5 +112,41 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 subjects:
   - name: cnrm-controller-manager-policies
+    namespace: cnrm-system
+    kind: ServiceAccount
+---
+# IAM Permissions to create folder sinks
+# Grant viewer role on the hierarchy namespace to config-control K8S SA
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata: # kpt-merge: hierarchy/allow-hierarchy-resource-reference-from-config-control
```

Comment:
the `kpt-merge` comment should be removed

Reply:
Fixed, please resolve the comment if satisfied.